lib/archethic/self_repair/sync.ex

```diff
@@ -276,7 +276,7 @@ defmodule Archethic.SelfRepair.Sync do
     Logger.info("Need to synchronize #{Enum.count(transaction_summaries)} transactions")
     Logger.debug("Transaction to sync #{inspect(transaction_summaries)}")
 
-    Task.Supervisor.async_stream_nolink(
+    Task.Supervisor.async_stream(
```

Comment:
It will crash the parent process, if downloading fails. process_transaction will only return error. Crash parent process to stop synchronisation of  transaction summary, and any  further, if downloading of anyone summary fails.

Reply:
The synchronisation will be stopped and the last sync date would not be updated as here: https://github.com/archethic-foundation/archethic-node/blob/master/lib/archethic/self_repair/scheduler.ex#L119